.dockerignore

```diff
@@ -0,0 +1,11 @@
+.vscode
+.idea
+*.iml
+.DS_Store
+.git
+
```

Comment:
We may add
```
Jenkinsfile
base-image-build.pipeline.groovy
for_devs_on_macos
scripts
```
This should improve docker cache handling, not strictly necessary, but I think it may help to avoid unnecessary builds.

Reply:
fixed